.github/workflows/osl_publish_images_snapshots.yml

```
@@ -46,6 +46,14 @@ jobs:
       - name: "Setup environment"
         uses: ./.github/actions/setup-env
 
+      - name: "Cache Maven repository"
+        uses: actions/cache@v3
```

Maybe is better to use the last version?
https://github.com/marketplace/actions/cache

```suggestion
        uses: actions/cache@v4
```